git_machete/cmd.py

```diff
@@ -2628,7 +2641,7 @@ def usage(c: str = None) -> None:
         "log": "Log the part of history specific to the given branch",
         "reapply": "Rebase the current branch onto its computed fork point",
         "show": "Show name(s) of the branch(es) relative to the position of the current branch, accepts down/first/last/next/root/prev/up argument",
-        "slide-out": "Slide out the current branch and sync its downstream (child) branch with its upstream (parent) branch via rebase or merge",
+        "slide-out": "Slide out the current branch and sync its downstream (child) branches with its upstream (parent) branch via rebase or merge",
```

Comment:
If you could also reflect this new change of logic in completion scripts! coz as for now, e.g. `develop` doesn't complete in the below scenario:

![image](https://user-images.githubusercontent.com/3383210/119534667-740a5180-bd87-11eb-9449-06e95a82242f.png)

Reply:
Also, out of curiosity, do you use the bash/zsh completion scripts? did you have any trouble setting them up?